servicebus/azure-messaging-servicebus/servicebus/pom.xml

```diff
@@ -41,7 +33,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-messaging-servicebus</artifactId>
-      <version>7.4.0</version> <!-- {x-version-update;com.azure:azure-messaging-servicebus;dependency} -->
+      <version>7.4.1</version> <!-- {x-version-update;com.azure:azure-messaging-servicebus;dependency} -->
```

Comment:
azure-sdk-bom already contain this.

Reply:
so you suggest to add azure-sdk-bom?

Reply:
azure-spring-boot-bom already imported azure-sdk-bom, and we already have azure-spring-boot-bom, so no need to import azure-sdk-bom.